/controller_node:
  ros__parameters:
    HEADING_KP: 0.01111111111
    TURNING_KP: 0.05

/driving_node:
  ros__parameters:
    i2c_addr: 0x40
    left_port_num: 0
    right_port_num: 3
